work_queue/src/client.py

```diff
@@ -0,0 +1,35 @@
+import sys
```

Comment:
Please add the standard CCTools copyright boilerplate (now 2020!) to the top of each file.